src/main/java/Command.java

```diff
@@ -0,0 +1,68 @@
+public class Command {
+    public static void findCommand(String taskInput){
+        String[] splitTaskInput = taskInput.toLowerCase().split(" ");
+        String[] taskDescription;
+        switch (splitTaskInput[0]){
+            case "done":
+                int taskNumberCompleted = Integer.parseInt(splitTaskInput[1]);
+                taskNumberCompleted--;
+                Task.completeTask(taskNumberCompleted);
+                UI.taskCompleted();
+                break;
+            case "deadline":
+                taskDescription = findTaskDescription(splitTaskInput);
+                Task.addNewTask(new Deadline(taskDescription[0],taskDescription[1]));
+                UI.taskAdded();
+                break;
+            case "event":
+                taskDescription = findTaskDescription(splitTaskInput);
+                Task.addNewTask(new Event(taskDescription[0],taskDescription[1]));
+                UI.taskAdded();
+                break;
+            case "todo":
+                taskDescription = findTaskDescription(splitTaskInput);
+                Task.addNewTask(new Todo(taskDescription[0]));
+                UI.taskAdded();
+                break;
+            case "list":
+                System.out.println("Here are the tasks in your list:");
+                for (int i = 0; i < Task.getTaskNumber(); i++){
+                    int displayedTask = i + 1;
+                    System.out.println(displayedTask + ": " + Task.getTaskList()[i]);
+                }
+                break;
+            default:
+                System.out.println("Huh? What do you mean? Please input a valid command word and task.");
+                break;
+        }
+    }
+
+    public static String[] findTaskDescription(String[] splitTaskInput){
+        String[] taskDescription = new String[2];
+        int slashIndex=1;
+        StringBuilder description = new StringBuilder();
+        StringBuilder deadline = new StringBuilder();
+        String space = "";
+        for (int i = 1; i<splitTaskInput.length;i++){
```

Comment:
Have you considered using String.split() method for finding the task description?